hack/lib/etcd.sh

```diff
@@ -23,7 +23,8 @@ ETCD_PORT=${ETCD_PORT:-2379}
 kube::etcd::validate() {
   # validate if in path
   command -v etcd >/dev/null || {
-    kube::log::usage "etcd must be in your PATH"
+    kube::log::info "You can use 'hack/install-etcd.sh' to install a copy in third_party/."
```

Comment:
nit: maybe switch these two lines? (i.e. print "etcd must be in your PATH" first)?

otherwise this LGTM.

Reply:
Done 👍